Weather Man/weatherman.py

```diff
@@ -0,0 +1,56 @@
+# weatherman.py
+from file_parser import WeatherParser
+from weather_calculator import WeatherCalculator
+from weather_reports import WeatherReport
+import sys
+
+
+def main():
+    args = sys.argv[1:]
+    if len(args) < 2:
+        print("Usage: weatherman.py /path/to/files-dir -e 2002")
+        sys.exit(1)
+
+    files_dir = args[0]
+    options = args[1:]
+
+    parser = WeatherParser(files_dir)
+
+    option_index = 0
+    while option_index < len(options):
+        option = options[option_index]
+        if option == '-e':
+            year = int(options[option_index + 1])
+
+            # get the weather data for the year
```

Comment:
No comments allowed, your class/method/variable name should represent the meaning